src/js/views/patients/patient/dashboard/dashboard_views.js

```diff
@@ -13,6 +14,15 @@ import LayoutTemplate from './layout.hbs';
 import '../patient.scss';
 import './dashboard.scss';
 
+const EmptyView = View.extend({
+  tagName: 'tr',
+  template: hbs`
+    <td class="patient-empty-list__body">
+      <h2 class="patient-empty-list__sub-heading">{{ @intl.patients.patient.dataEvents.dataEventsViews.emptyView }}</h2>
+    </td>
+  `,
+});
+
```

Comment:
this one doesn't actually show up because the 'new action' template is always there (and I think should only be there after clicking 'add action' or whatever and should go away if you close the sidebar w/o adding an action?)

Reply:
I don't follow.  The new action template only shows up for me after pushing add action, and does go away when I close the new action sidebar (if it's open)

Reply:
🤔 maybe it was just the testing env I had setup that was wonky